src/topology/uniform_space/uniform_convergence_topology.lean

```diff
@@ -499,6 +553,70 @@ begin
   refl
 end
 
+private lemma has_basis_uniformity_of_basis_aux₁ {p : ι → Prop} {s : ι → set (β × β)}
```

Comment:
```suggestion
lemma has_basis_uniformity_of_basis_aux₁ {p : ι → Prop} {s : ι → set (β × β)}
```
I prefer to have lemmas not be private, because metaprogramming functions cannot find private definitions, and jump-to-definition doesn't work for them. Also below.